### ISSUE TITLE
Live feed from webcam

### ISSUE THREAD

**Issue by shahanhyperspace** [kijai/ComfyUI-LivePortraitKJ#138]
Is it possible to connect a webcam input in and generate the out at runtime? Rather than recording a video and uploading it.

**Comment by kijai** [comment]
Yes there's a realtime example included, it captures a part of your screen which can be anything, including a view from your webcam. There are nodes to capture webcam directly too, but every one I tried was too slow.

**Comment by DenizYunus** [comment]
Well, I'm new to ComfyUI, and I couldn't find a way to run the webcam example continuously. I just can press queue, and it generates 1 image based on current webcam frame. As a developer, I thought I could use sth like Loops but couldn't find that too. Tried some nodes like loopchain etc but couldn't make them work too

Update: I used Auto Queue, it works actually so no need to loop functionality.

**Comment by rickstaa** [comment]
You may have already come across this, but my team put together this [custom node](https://docs.comfystream.org/technical/get-started/introduction) that	allows you to stream video and audio into ComfyUI in real-time. Hopefully, it’s useful to someone!
